File: app/src/main/java/com/ephirium/storyline/ui/main/fragment/DescriptionStoryFragment.java
package com.ephirium.storyline.ui.main.fragment;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.ephirium.storyline.databinding.FragmentDescriptionStoryBinding;

public class DescriptionStoryFragment extends Fragment {

    private FragmentDescriptionStoryBinding binding;

    public DescriptionStoryFragment() {
        // default
    }

    public static DescriptionStoryFragment newInstance(){
        return new DescriptionStoryFragment();
    }

    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        binding = FragmentDescriptionStoryBinding.inflate(getLayoutInflater());
        return binding.getRoot();
    }
}

File: app/src/main/java/com/ephirium/storyline/ui/fragment/DescriptionStoryFragment.kt
package com.ephirium.storyline.ui.fragment;


import android.os.Bundle
import android.view.View
import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import by.kirich1409.viewbindingdelegate.viewBinding
import com.ephirium.storyline.R

import com.ephirium.storyline.databinding.FragmentDescriptionStoryBinding;

class DescriptionStoryFragment : Fragment(R.layout.fragment_description_story) {

    private val binding: FragmentDescriptionStoryBinding by viewBinding()

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
    }
}